generators/app/generate.php

```diff
@@ -35,6 +36,12 @@ function generate() {
 
     $this->unzipAndRemoveTemplateZip();
 
+    // Whippet deploy requires at least one commit in the repo.
```

Comment:
Non-blocking suggestion: Would it be worth moving this slightly further down so that it happens after the permissions on the `script/` files have been fixed? 

This might also be a good time to remove the call to `$this->setWpVersion();`, as out default use case these days would be to pin to the major version set in the template repo.